src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -111,6 +118,9 @@ def load_arguments(self, _):
         c.argument('is_incremental', action='store_true', options_list=['--is-incremental', '-i'],
                    help='The option to add tags incrementally without deleting the original tags. If the key of new tag and original tag are duplicated, the original value will be overwritten.')
 
+    with self.argument_context('resource wait') as c:
+        c.ignore('latest_include_preview')
```

Comment:
I don't think we need this one

Reply:
Because command `resource wait` cannot refer to parameter `latest_include_preview`:
https://github.com/Azure/azure-cli/blob/bdbd6e07f8dd3f0d9a8b943269ebea60b7afe7d3/src/azure-cli/azure/cli/command_modules/resource/commands.py#L196